app/notifications/logbook_entry_updated_notification.rb

```diff
@@ -0,0 +1,32 @@
+
+class LogbookEntryUpdatedNotification < Noticed::Base
+  deliver_by :database, if: :database_notifications?
+  deliver_by :email, mailer: "LogbookEntryMailer", method: "updated_email", if: :email_notifications?
+
+  DEFAULT_SEND_NOTIFICATION = true
+  DEFAULT_SEND_EMAIL_NOTIFICATION = true
+
+  param :logbook_entry
+
+  def message
+    aircraft_model = Aircraft.find_by(id: params[:logbook_entry].aircraft_id).model
+    "#{params[:logbook_entry].first_name}'s #{params[:logbook_entry].date} logbook entry has been updated for their flight sim on the #{aircraft_model}."
```

Comment:
```suggestion
    "#{params[:logbook_entry].first_name}'s #{params[:logbook_entry].date} logbook entry has been updated for their flight sim on the #{params[:logbook_entry].aircraft}."
```


Then in the `aircraft.rb` model:
```ruby
def to_s
  make + model
end